### PR TITLE
Set number of mappers to either the specified or the number of files

### LINKED ISSUE #83: Add a math.min for the number of mappers based on the number of files
ambiata/view#36

### PR DESCRIPTION
**Pull request by nhibberd**
Fixes #83

### CODE REVIEW

**Comment by nhibberd** [comment]
@markhibberd

File: notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala [hunk 1]
```diff
@@ -43,7 +43,8 @@ object DistCopyJob {
       job.getConfiguration.setInt(ReadLimit, conf.readLimit.toBytes.value.toInt)
       job.getConfiguration.setLong(MultipartUploadThreshold, conf.multipartUploadThreshold.toBytes.value)
     })
-    _   <- DistCopyInputFormat.setMappings(job, ctx, conf.client, mappings, conf.mappersNumber)
+    n   = Math.min(mappings.mappings.length, conf.mappersNumber)
```
Comment (markhibberd) [inline notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala]: Need to guard against zero length mappings.

**Comment by nhibberd** [comment]
@raronson @domdere bumps

**Comment by domdere** [comment]
I dont see any problems :+1:

**Comment by raronson** [comment]
:+1:

**Pull request merged**